dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.wss4j-2.3.feature

```diff
@@ -18,5 +18,5 @@ singleton=true
  com.ibm.ws.org.joda.time.2.9.9, \
  com.ibm.ws.org.opensaml.opensaml.core.3.4.5.jakarta, \
  com.ibm.ws.net.shibboleth.utilities.java.support.7.5.1.jakarta
-kind=noship
-edition=full
+kind=beta
```

Comment:
Not sure why this is moving to beta when other things aren't.

Reply:
So now this private features isn't referenced by anything any longer.  Should it be deleted then?

Reply:
moved it back to noship

Reply:
this is referenced by io.openliberty.wsSecurity1.1.internal.jaxws-3.0.feature (wssecurity requires this)

Reply:
yeah I removed my comment about it not being referenced when I saw that.